lib/pageflow/public_i18n/version.rb

```diff
@@ -1,5 +1,5 @@
 module Pageflow
   module PublicI18n
-    VERSION = '1.19.0'.freeze
+    VERSION = '1.19.1'.freeze
```

Comment:
Adding new keys means new features, which according to SemVer requires a new minor version. I think it's best practice, though, to alter things like the version number or the changelog during release only to prevent merge conflicts with other PRs.

Reply:
reverted the version